InvenTree/build/templates/build/detail.html

```diff
@@ -337,11 +360,9 @@ <h4>{% trans "Build Notes" %}</h4>
             build: {{ build.id }},
             is_building: false,
         },
-        groupByField: 'location',
```

Comment:
Question: Will this change the default behavior and is that intended?
Suggestion: If yes: Should this PR be marked as breaking?

Reply:
No, stock-item grouping has long been removed! This is just a cleanup of legacy code